prebid-mobile/pbm-api/android/code-integration-android.md

```diff
@@ -96,15 +89,17 @@ If you want to define a different Account Settings ID as determined in conjuncti
 your Prebid Server team, use the [arbitrary OpenRTB](/prebid-mobile/pbm-api/android/pbm-targeting-android.html#arbitrary-openrtb) method like this:
 
 ```kotlin
-adUnitConfiguration?.ortbConfig = "{\"ext\":{\"prebid\":{\"storedrequest\": {\"id\":\"account-settings-id\"}}}}"
+adUnit?.impOrtbConfig = "{\"ext\":{\"prebid\":{\"storedrequest\": {\"id\":\"account-settings-id\"}}}}"
```

Comment:
This isn't right -- account-settings is not at the AdUnit level. account-settings is the global ext.prebid.storedrequest.